monarchmoney/monarchmoney.py

```diff
@@ -880,6 +880,168 @@ async def get_cashflow_summary(
             operation="Web_GetCashFlowPage", variables=variables, graphql_query=query
         )
 
+    async def update_transaction(
+        self,
+        transaction_id: str,
+        category_id: Optional[str] = None,
+        merchant_name: Optional[str] = None,
```

Comment:
From within the web app, I attempted to test the length limits for merchant_name and note. I gave up after getting to 10k+ characters for both! I don't know if it's worth searching further (for the sake of documentation - or adding a guard) but just writing this as an FYI.